cmd/pool/order.go

```diff
@@ -116,6 +116,20 @@ var sharedFlags = []cli.Flag{
 			channelTypePeerDependent,
 			channelTypeScriptEnforced),
 	},
+	cli.StringSliceFlag{
+		Name: "allowed_node_id",
+		Usage: "the list of nodes this order is able to match with; " +
+			"if empty, the order will be able to match with any " +
+			"node unless not_allowed_node_id is set. Cann be " +
+			"specified multiple times",
+	},
+	cli.StringSliceFlag{
+		Name: "not_allowed_node_id",
+		Usage: "the list of nodes this order is not able to match " +
```

Comment:
nit: s/not able to match with/not allowed to match with? "not able" implies a problem, but this is just a user preference.